### ISSUE TITLE
Runner for windows/arm64 platforms

### ISSUE THREAD

**Issue by nsait-linaro** [actions/runner#1358]
We are looking at adding CI/CD support for a couple of open source projects to be able to build and tests on windows/arm64 targets using self-hosted runners.

As most of the open-source projects use GitHub actions we wouldn't be able to add CI/CD support for them for win/arm64 machines without runner support on the target machines.

If Win/arm64 build is not straightforward, even an x86 build of runner would be helpful as we can run runner in an x86 emulation mode from win/arm64 machines.

**Comment by ethomson** [comment]
Thanks @nsait-linaro - I'll put this on the backlog.

**Comment by camptatopenmars** [comment]
I have found that the windows x64 runner package works just fine on current win10 arm64 builds (which now support x64 binaries) with one problem: it ends up being tagged with 'X64' and I had to manually add my own 'ARM64' tag to it.  As such I had to change all of actual X64 runners and actions to have/use an additional 'REALX64' tag to correctly distinguish architectures.  But otherwise works fine.

I need an windows x86 runner as well since I'm testing a driver and the kernel has to actually match the driver architecture.
